redis.conf

```diff
@@ -1449,6 +1449,12 @@ lua-time-limit 5000
 #
 # cluster-node-timeout 15000
 
+# Cluster port incr is the offset from command port to cluster bus port. 
+# Cluster port = command port + cluster port incr.
+# User could define the difference from them. 
+#
+# cluster-port-incr 10000
```

Comment:
Instead, can I propose we just make this a new parameter called cluster-port ? If it's not defined we can derive the value like we do today. I know it adds a bit of complexity, but it makes a lot more sense for a user controlled config.

Reply:
Hi Madolson, if we change the parameter name to cluster-port, that will be another program logic. In this pr, the new parameter name "cluster-port-incr" is mapping the marco definition "#define CLUSTER_PORT_INCR 10000 in cluster.h".
The parameter means the offset (aka difference between command port and cluster bus port).

If you think we could give it a better name, we could name it as cluster-port-offset or similar,  how do you think?

Reply:
What I'm arguing is that there is no reason to be doing this based off of an offset. We don't gossip the offset, we gossip the real port number. Having the +10000 offset is nice from simplicity sake, but in reality most people would want to set the cluster port specifically, and not reverse engineer the offset needed.

Reply:
I got your point, let me change this parameter to allow people to set the cluster bus port.  Thanks for your explanation.